src/Symfony/Component/Cache/Adapter/DoctrineDbalAdapter.php

```diff
@@ -339,7 +339,7 @@ protected function doSave(array $values, int $lifetime)
      */
     protected function getId($key)
     {
-        if ('pgsql' !== $this->platformName ??= $this->getPlatformName()) {
+        if ('pgsql' !== $this->getPlatformName()) {
```

Comment:
`$this->getPlatformName()` already memoizes its return value on `$this->platformName`, so I don't see a reason to shortcut that anyway.